app/javascript/components/upload/ExpandableFileForm.jsx

```diff
@@ -3,17 +3,22 @@ import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
 import { faChevronDown, faChevronUp, faTimes } from '@fortawesome/free-solid-svg-icons'
 import Modal from 'react-bootstrap/lib/Modal'
 import { Popover, OverlayTrigger } from 'react-bootstrap'
-
+import { clusterFileFilter } from './ClusteringStep'
+import { metadataFileFilter } from './MetadataStep'
+import { rawCountsFileFilter } from './RawCountsStep'
+import { processedFileFilter } from './ProcessedExpressionStep'
 import LoadingSpinner from '~/lib/LoadingSpinner'
 import FileUploadControl from './FileUploadControl'
 
 /** renders its children inside an expandable form with a header for file selection */
 export default function ExpandableFileForm({
   file, allFiles, updateFile, allowedFileExts, validationMessages, bucketName,
-  saveFile, deleteFile, isInitiallyExpanded, children
+  saveFile, deleteFile, isInitiallyExpanded, isAnnDataExperience, children
 }) {
   const [expanded, setExpanded] = useState(isInitiallyExpanded || file.status === 'new')
 
+  const doNotShowFileUploadControls = displayFileUploadControls(isAnnDataExperience, allFiles)
```

Comment:
See related comments.
```suggestion
  const isUploadEnabled = displayFileUploadControls(isAnnDataExperience, allFiles)
```
```suggestion
  const doNotShowFileUploadControls = displayFileUploadControls(isAnnDataExperience, allFiles)
```